src/hooks/scrapers.js

```diff
@@ -4,10 +4,10 @@ import createRequestObject from './createRequest'
 
 export async function stockxLowestPrice (item, filter) {
   try {
-    const modelName = item.modelName.replace('(W)', '')
+    const model = item.model.replace('(W)', '')
```

Comment:
Why not call this `model_name`? I think it's clearer (since model could be a lot of things) and its consistent with the backend.

Reply:
I actually changed it to `model` to be consistent with what the browse endpoint returns (if you're referring to the item field not the variable name). Before this change we were manually deconstructing the item object in the frontend to match the backend fields which was no bueno.